src/Resources/config/makers.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+        <services>
+            <defaults public="false" />
+
+            <service id="maker.maker.make_authenticator" class="Symfony\Bundle\MakerBundle\Maker\MakeAuthenticator">
```

Comment:
Unimportant proposal: I'd love to use autowiring ... but if it's not possible, could we use the class as the service ID? ... if that's not possible either, could we rename the IDs to `maker.command.*` (the `maker.maker.*` format looks too redundant).

Reply:
So.... this is done on purpose :)

* If third-party bundles use autowiring, then it will slow the dev experience for users a little bit. I'd like to make it a standard to *not* use autowiring for this reason.

* If we use the class names as service id's, they will show up in `debug:autowiring`. We need to tell the bundle world to not do this - only use class names or class aliases *if* it is something that makes sense to autowire.

* The `maker.command.*` is a little misleading... as these aren't actually commands, but are... "makers"... whatever that means ;).